recipes/patsy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.4.1" %}
+{% set gh_org = "pydata" %}
+{% set gh_repo = "patsy" %}
+{% set pkg_name = "patsy" %}
```

Comment:
I know you didn't add these, @croth1, but I agree. Let's get rid of `gh_org` and `gh_repo` as they are really only used once each in the home page. It is unclear what value there is in templating the home page as it could just as easily not be on GitHub.